Cannot build using CMake + gcc 14 on Fedora Linux

### Describe the bug

I am unable to build the game on Fedora Linux using CMake and gcc 14.

https://github.com/cataclysmbnteam/Cataclysm-BN/blob/92be5ba4ac691d4f8b3a858d6b7c5762f693dca3/CMakeLists.txt#L258-L260

In particular, it appears to be ignoring the instructions in CMakeLists.txt to build with c++17, and is instead trying to use c++20. Due to this, errors occur.

### Steps To Reproduce

1. Follow the instructions on the docs page for building with CMake
`cmake .. -DCMAKE_BUILD_TYPE=Release -DTiles=ON -DSOUND=ON`
3. Attempt to `make`
4. Watch it burn almost immediately

### Screenshots

![image](https://github.com/user-attachments/assets/29384335-4dab-4891-b28e-110ce2c96269)
![image](https://github.com/user-attachments/assets/8b615954-8ec0-4ab7-bba5-ddb06c1d30b4)


### Versions and configuration

Fedora 40 KDE, Linux
gcc 14.1.1
CMake 3.28.2

### Additional context

Issue on Github was requested by Scarf

1. Starting with a freshly downloaded copy of the source code (for the sake of making sure it wasn't just my previous copy acting strange):
```
rosaaeterna@fedora:~/Documents/Cataclysm-BN-2024-07-11$ mkdir build && cd build
rosaaeterna@fedora:~/Documents/Cataclysm-BN-2024-07-11/build$ cmake .. -DCMAKE_BUILD_TYPE=Release -DTILES=ON -DSOUND=ON
-- The C compiler identification is GNU 14.1.1
-- The CXX compiler identification is GNU 14.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Artistic style executable was not found, so automatic code style formating will be unavailable
-- CataclysmBN build environment --
-- Build realm is: Linux x86_64
-- CataclysmBN build version is: HEAD-HASH
fatal: not a git repository (or any of the parent directories): .git
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.1.0") 
-- CataclysmBN build options --
-- Found Git: /usr/bin/git (found version "2.45.2") 
-- CMAKE_INSTALL_PREFIX          : /usr/local
-- CMAKE_INSTALL_BINDIR          : 
-- DESKTOP_ENTRY_PATH            : 
-- PIXMAPS_ENTRY_PATH            : 
-- PIXMAPS_UNITY_ENTRY_PATH      : 
-- MANPAGE_ENTRY_PATH            : 
-- GIT_BINARY                    : /usr/bin/git
-- DYNAMIC_LINKING               : ON
-- TILES                         : ON
-- CURSES                        : OFF
-- SOUND                         : ON
-- LUA                           : OFF
-- BACKTRACE                     : ON
-- LIBBACKTRACE                  : OFF
-- USE_TRACY                     : OFF
-- USE_XDG_DIR                   : OFF
-- USE_HOME_DIR                  : ON
-- UNITY_BUILD                   : OFF
-- LANGUAGES                     : all
-- See CMake compiling guide for details and more info --
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Searching for SDL2 library --
-- Searching for SDL2_TTF library --
-- Found SDL2_ttf: /usr/lib64/libSDL2_ttf.so  
-- Searching for SDL2_image library --
-- Found SDL2_image: /usr/lib64/libSDL2_image.so  
-- Searching for SDL2_mixer library --
-- Found SDL2_mixer: /usr/lib64/libSDL2_mixer.so  
-- Searching for SDL_mixer deps libraries --
-- Checking for module 'flac'
--   Found flac, version 1.4.3
-- Found Gettext: /usr/bin/msgmerge (found version "0.22.5") 
-- Process LANGUAGES variable --



-- Configuring done (4.0s)
-- Generating done (2.3s)
-- Build files have been written to: /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build
rosaaeterna@fedora:~/Documents/Cataclysm-BN-2024-07-11/build$ make -j 4
[  1%] Built target get_version
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx.gch
[  1%] Built target translations_prepare
[  1%] Built target translations_compile
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_tile_data.cpp:8:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_item_cache.cpp:6:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/inventory.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/character.h:36,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/avatar.h:12,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/action.cpp:11:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/inventory.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/character.h:36,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/achievement.cpp:9:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:125: src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:141: src/CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o] Error 1
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:109: src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o] Error 1
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:93: src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1017: src/CMakeFiles/cataclysm-tiles-common.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

2. GCC 13.3 is more easily accessible to me, would that also be acceptable?

```
rosaaeterna@fedora:~/Documents/Cataclysm-BN-2024-07-11/build$ make VERBOSE=1 -j 4
/usr/bin/cmake -S/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 -B/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/CMakeFiles /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/depend
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/lang /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/lang/CMakeFiles/translations_prepare.dir/DependInfo.cmake "--color="
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/get_version.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/build
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ar/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make[2]: Nothing to be done for 'src/CMakeFiles/get_version.dir/build'.
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/da/LC_MESSAGES
[  1%] Built target get_version
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/depend
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/de/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/DependInfo.cmake "--color="
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/el/LC_MESSAGES
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/es_AR/LC_MESSAGES
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx.gch.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx.gch.d" is newer than depends file "/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Consolidate compiler generated dependencies of target cataclysm-tiles-common
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/build
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/es_ES/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/fr/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o -c /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_item_cache.cpp
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/hu/LC_MESSAGES
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o -c /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/action.cpp
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o -c /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/achievement.cpp
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/id/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/is/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/it_IT/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ja/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ko/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/nb/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/nl/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/pl_PL/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/pt_BR/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ru_RU/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/sr/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/tr/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/uk/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/uk_UA/LC_MESSAGES
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/zh_CN/LC_MESSAGES
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_item_cache.cpp:6:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/zh_TW/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
[  2%] Built target translations_prepare
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o -c /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_tile_data.cpp
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/inventory.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/character.h:36,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/avatar.h:12,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/action.cpp:11:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/inventory.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/character.h:36,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/achievement.cpp:9:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
In file included from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/item.h:18,
                 from /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/active_tile_data.cpp:8:
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   40 |         cata_arena<T>() = default;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:40:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   41 |         cata_arena<T>( const cata_arena<T> & ) = delete;
      |                      ^
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:41:22: note: remove the ‘< >’
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: error: template-id not allowed for constructor in C++20 [-Werror=template-id-cdtor]
   42 |         cata_arena<T>( cata_arena<T> && ) = delete;
      |                        ^~~~~~~~~~~~~
/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/src/cata_arena.h:42:24: note: remove the ‘< >’
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:125: src/CMakeFiles/cataclysm-tiles-common.dir/active_item_cache.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/lang /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build/lang/CMakeFiles/translations_compile.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/ar.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ar/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/da.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/da/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/de.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/de/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/el.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/el/LC_MESSAGES/cataclysm-bn.mo
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:141: src/CMakeFiles/cataclysm-tiles-common.dir/active_tile_data.cpp.o] Error 1
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/es_AR.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/es_AR/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/es_ES.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/es_ES/LC_MESSAGES/cataclysm-bn.mo
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:109: src/CMakeFiles/cataclysm-tiles-common.dir/action.cpp.o] Error 1
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/fr.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/fr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/hu.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/hu/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/id.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/id/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/is.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/is/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/it_IT.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/it_IT/LC_MESSAGES/cataclysm-bn.mo
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:93: src/CMakeFiles/cataclysm-tiles-common.dir/achievement.cpp.o] Error 1
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make[1]: *** [CMakeFiles/Makefile2:1017: src/CMakeFiles/cataclysm-tiles-common.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/ja.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ja/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/ko.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ko/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/nb.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/nb/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/nl.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/nl/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/pl_PL.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/pl_PL/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/pt_BR.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/pt_BR/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/ru_RU.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/ru_RU/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/sr.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/sr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/tr.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/tr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/uk.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/uk/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/uk_UA.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/uk_UA/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/zh_CN.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/zh_CN/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11 && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/po/zh_TW.po -o /home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/lang/mo/zh_TW/LC_MESSAGES/cataclysm-bn.mo
make[2]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
[  2%] Built target translations_compile
make[1]: Leaving directory '/home/rosaaeterna/Documents/Cataclysm-BN-2024-07-11/build'
make: *** [Makefile:146: all] Error 2
```

Have some verbose output, evidently passing that one doesn't work
```
rosaaeterna@fedora:~/Documents/GitHub/Cataclysm-BN/build$ CXXFLAGS=-Wno-array-bounds make VERBOSE=1 -j 4
/usr/bin/cmake -S/home/rosaaeterna/Documents/GitHub/Cataclysm-BN -B/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/CMakeFiles /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/depend
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang/CMakeFiles/translations_prepare.dir/DependInfo.cmake "--color="
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/get_version.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ar/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[2]: Nothing to be done for 'src/CMakeFiles/get_version.dir/build'.
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/da/LC_MESSAGES
[  1%] Built target get_version
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/de/LC_MESSAGES
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/depend
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/el/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_AR/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/DependInfo.cmake "--color="
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_ES/LC_MESSAGES
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ammo.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ammo.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ammo_effect.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ammo_effect.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/anatomy.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/anatomy.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/animation.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/animation.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/armor_layers.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/armor_layers.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Consolidate compiler generated dependencies of target cataclysm-tiles-common
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/fr/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/build
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/hu/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/id/LC_MESSAGES
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/is/LC_MESSAGES
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/ascii_art.cpp
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/artifact.cpp
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/it_IT/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ja/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ko/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nb/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nl/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pl_PL/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pt_BR/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ru_RU/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/sr/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/tr/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk_UA/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_CN/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_TW/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
[  1%] Built target translations_prepare
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/assign.cpp
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/auto_note.cpp
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang/CMakeFiles/translations_compile.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ar.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ar/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/da.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/da/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/de.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/de/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/el.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/el/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/es_AR.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_AR/LC_MESSAGES/cataclysm-bn.mo
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/auto_pickup.cpp
In file included from /usr/include/c++/14/bits/stl_tempbuf.h:61,
                 from /usr/include/c++/14/bits/stl_algo.h:69,
                 from /usr/include/c++/14/algorithm:61,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/pch/main-pch.hpp:2,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx:5,
                 from <command-line>:
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  119 |       ::new((void*)__p) _Tp(std::forward<_Args>(__args)...);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.h:6,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1:
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:276:28: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  276 |           this->_M_engaged = true;
      |           ~~~~~~~~~~~~~~~~~^~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:313:19: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  313 |         if (this->_M_engaged)
      |             ~~~~~~^~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:316:28: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  316 |           this->_M_engaged = false;
      |           ~~~~~~~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_destroy() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:314:14,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:282:20: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  282 |         _M_engaged = false;
      |         ~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_destroy() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:314:14,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:283:9: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  283 |         _M_payload._M_value.~_Stored_type();
      |         ^~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/es_ES.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_ES/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/fr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/fr/LC_MESSAGES/cataclysm-bn.mo
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/avatar.cpp
In file included from /usr/include/c++/14/vector:66,
                 from /usr/include/c++/14/functional:64,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/pch/main-pch.hpp:28:
In member function ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1608:24: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1608 |       { _M_erase_at_end(this->_M_impl._M_start); }
      |         ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘void std::vector<_Tp, _Alloc>::_M_erase_at_end(pointer) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1608:24,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1945:43: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1945 |         if (size_type __n = this->_M_impl._M_finish - __pos)
      |                             ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘void std::vector<_Tp, _Alloc>::_M_erase_at_end(pointer) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1608:24,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1949:37: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1949 |             this->_M_impl._M_finish = __pos;
      |             ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /usr/include/c++/14/vector:72:
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:114:27: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  114 |         if (this->_M_impl._M_finish != this->_M_impl._M_end_of_storage)
      |             ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:114:54: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  114 |         if (this->_M_impl._M_finish != this->_M_impl._M_end_of_storage)
      |                                        ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:119:29: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  119 |             ++this->_M_impl._M_finish;
      |               ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:119:13: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  119 |             ++this->_M_impl._M_finish;
      |             ^~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /usr/include/c++/14/bits/stl_algobase.h:67,
                 from /usr/include/c++/14/algorithm:60:
In constructor ‘__gnu_cxx::__normal_iterator<_Iterator, _Container>::__normal_iterator(const _Iterator&) [with _Iterator = iuse_location*; _Container = std::vector<iuse_location>]’,
    inlined from ‘std::vector<_Tp, _Alloc>::iterator std::vector<_Tp, _Alloc>::end() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:894:16,
    inlined from ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::back() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1238:14,
    inlined from ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/vector.tcc:125:13,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_iterator.h:1068:9: error: array subscript 1 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1068 |       : _M_current(__i) { }
      |         ^~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/hu.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/hu/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/id.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/id/LC_MESSAGES/cataclysm-bn.mo
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/is.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/is/LC_MESSAGES/cataclysm-bn.mo
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:157: src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/it_IT.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/it_IT/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ja.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ja/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ko.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ko/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/nb.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nb/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/nl.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nl/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/pl_PL.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pl_PL/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/pt_BR.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pt_BR/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ru_RU.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ru_RU/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/sr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/sr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/tr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/tr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/uk.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/uk_UA.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk_UA/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/zh_CN.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_CN/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/zh_TW.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_TW/LC_MESSAGES/cataclysm-bn.mo
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[1]: *** [CMakeFiles/Makefile2:1017: src/CMakeFiles/cataclysm-tiles-common.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
[  2%] Built target translations_compile
make[1]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make: *** [Makefile:146: all] Error 2
```

Here's it with the flag put after make; nope.
```
rosaaeterna@fedora:~/Documents/GitHub/Cataclysm-BN/build$ make VERBOSE=1 CXXFLAGS=-Wno-array-bounds -j 4
/usr/bin/cmake -S/home/rosaaeterna/Documents/GitHub/Cataclysm-BN -B/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/bin/cmake -E cmake_progress_start /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/CMakeFiles /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build//CMakeFiles/progress.marks
make  -f CMakeFiles/Makefile2 all
make[1]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/depend
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang/CMakeFiles/translations_prepare.dir/DependInfo.cmake "--color="
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/get_version.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f src/CMakeFiles/get_version.dir/build.make src/CMakeFiles/get_version.dir/build
make  -f lang/CMakeFiles/translations_prepare.dir/build.make lang/CMakeFiles/translations_prepare.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ar/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[2]: Nothing to be done for 'src/CMakeFiles/get_version.dir/build'.
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
[  1%] Built target get_version
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/da/LC_MESSAGES
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/depend
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/de/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/DependInfo.cmake "--color="
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/el/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_AR/LC_MESSAGES
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/artifact.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/ascii_art.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/assign.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/auto_note.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/auto_pickup.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Dependencies file "src/CMakeFiles/cataclysm-tiles-common.dir/avatar.cpp.o.d" is newer than depends file "/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/compiler_depend.internal".
Consolidate compiler generated dependencies of target cataclysm-tiles-common
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_ES/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f src/CMakeFiles/cataclysm-tiles-common.dir/build.make src/CMakeFiles/cataclysm-tiles-common.dir/build
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/fr/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/hu/LC_MESSAGES
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/id/LC_MESSAGES
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/avatar_functions.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/avatar_functions.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/avatar_functions.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/avatar_functions.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/avatar_functions.cpp
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp
[  1%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/avatar_action.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/avatar_action.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/avatar_action.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/avatar_action.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/avatar_action.cpp
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/is/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/it_IT/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ja/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ko/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nb/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nl/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pl_PL/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pt_BR/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ru_RU/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/sr/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/tr/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk_UA/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_CN/LC_MESSAGES
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/cmake -E make_directory /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_TW/LC_MESSAGES
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
[  1%] Built target translations_prepare
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/ballistics.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/ballistics.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/ballistics.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/ballistics.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/ballistics.cpp
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/depend
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build && /usr/bin/cmake -E cmake_depends "Unix Makefiles" /home/rosaaeterna/Documents/GitHub/Cataclysm-BN /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/lang/CMakeFiles/translations_compile.dir/DependInfo.cmake "--color="
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make  -f lang/CMakeFiles/translations_compile.dir/build.make lang/CMakeFiles/translations_compile.dir/build
make[2]: Entering directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ar.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ar/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/da.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/da/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/de.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/de/LC_MESSAGES/cataclysm-bn.mo
In file included from /usr/include/c++/14/bits/stl_tempbuf.h:61,
                 from /usr/include/c++/14/bits/stl_algo.h:69,
                 from /usr/include/c++/14/algorithm:61,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/pch/main-pch.hpp:2,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx:5,
                 from <command-line>:
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  119 |       ::new((void*)__p) _Tp(std::forward<_Args>(__args)...);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In function ‘void std::_Construct(_Tp*, _Args&& ...) [with _Tp = tripoint; _Args = {}]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:274:19,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/bits/stl_construct.h:119:7: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.h:6,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1:
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_construct(_Args&& ...) [with _Args = {}; _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:455:52,
    inlined from ‘std::enable_if_t<((bool)is_constructible_v<_Tp, _Args ...>), _Tp&> std::optional<_Tp>::emplace(_Args&& ...) [with _Args = {}; _Tp = tripoint]’ at /usr/include/c++/14/optional:916:22,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1551:20,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1546:6,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:276:28: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  276 |           this->_M_engaged = true;
      |           ~~~~~~~~~~~~~~~~~^~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:313:19: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  313 |         if (this->_M_engaged)
      |             ~~~~~~^~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:316:28: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  316 |           this->_M_engaged = false;
      |           ~~~~~~~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_destroy() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:314:14,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:282:20: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  282 |         _M_engaged = false;
      |         ~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘constexpr void std::_Optional_payload_base<_Tp>::_M_destroy() [with _Tp = tripoint]’,
    inlined from ‘constexpr void std::_Optional_payload_base<_Tp>::_M_reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:314:14,
    inlined from ‘constexpr void std::_Optional_base_impl<_Tp, _Dp>::_M_reset() [with _Tp = tripoint; _Dp = std::_Optional_base<tripoint, true, true>]’ at /usr/include/c++/14/optional:466:53,
    inlined from ‘void std::optional<_Tp>::reset() [with _Tp = tripoint]’ at /usr/include/c++/14/optional:1180:66,
    inlined from ‘void deserialize(std::optional<_Tp>&, JsonIn&) [with T = tripoint]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1549:18,
    inlined from ‘decltype ((deserialize(v, (*(JsonIn*)this)), true)) JsonIn::read(T&, bool) [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:313:28,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::optional<tripoint>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1888:14:
/usr/include/c++/14/optional:283:9: error: array subscript 0 is outside array bounds of ‘std::optional<tripoint> [0]’ [-Werror=array-bounds=]
  283 |         _M_payload._M_value.~_Stored_type();
      |         ^~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> throw_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/el.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/el/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/es_AR.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_AR/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/es_ES.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/es_ES/LC_MESSAGES/cataclysm-bn.mo
In file included from /usr/include/c++/14/vector:66,
                 from /usr/include/c++/14/functional:64,
                 from /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/pch/main-pch.hpp:28:
In member function ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1608:24: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1608 |       { _M_erase_at_end(this->_M_impl._M_start); }
      |         ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘void std::vector<_Tp, _Alloc>::_M_erase_at_end(pointer) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1608:24,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1945:43: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1945 |         if (size_type __n = this->_M_impl._M_finish - __pos)
      |                             ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘void std::vector<_Tp, _Alloc>::_M_erase_at_end(pointer) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::clear() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1608:24,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:434:24,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_vector.h:1949:37: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1949 |             this->_M_impl._M_finish = __pos;
      |             ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /usr/include/c++/14/vector:72:
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:114:27: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  114 |         if (this->_M_impl._M_finish != this->_M_impl._M_end_of_storage)
      |             ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:114:54: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  114 |         if (this->_M_impl._M_finish != this->_M_impl._M_end_of_storage)
      |                                        ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:119:29: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  119 |             ++this->_M_impl._M_finish;
      |               ~~~~~~~~~~~~~~^~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In member function ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/vector.tcc:119:13: error: array subscript 0 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
  119 |             ++this->_M_impl._M_finish;
      |             ^~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
In file included from /usr/include/c++/14/bits/stl_algobase.h:67,
                 from /usr/include/c++/14/algorithm:60:
In constructor ‘__gnu_cxx::__normal_iterator<_Iterator, _Container>::__normal_iterator(const _Iterator&) [with _Iterator = iuse_location*; _Container = std::vector<iuse_location>]’,
    inlined from ‘std::vector<_Tp, _Alloc>::iterator std::vector<_Tp, _Alloc>::end() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:894:16,
    inlined from ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::back() [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1238:14,
    inlined from ‘std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::emplace_back(_Args&& ...) [with _Args = {iuse_location}; _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/vector.tcc:125:13,
    inlined from ‘void std::vector<_Tp, _Alloc>::push_back(value_type&&) [with _Tp = iuse_location; _Alloc = std::allocator<iuse_location>]’ at /usr/include/c++/14/bits/stl_vector.h:1301:21,
    inlined from ‘decltype ((v.front(), true)) JsonIn::read(T&, bool) [with T = std::vector<iuse_location>; typename std::enable_if<(! std::is_same<void, typename T::value_type>::value)>::type* <anonymous> = 0]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:438:36,
    inlined from ‘bool JsonObject::read(const std::string&, T&, bool) const [with T = std::vector<iuse_location>]’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/json.h:1036:30,
    inlined from ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’ at /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/activity_actor.cpp:1909:14:
/usr/include/c++/14/bits/stl_iterator.h:1068:9: error: array subscript 1 is outside array bounds of ‘iuse_locations [0]’ [-Werror=array-bounds=]
 1068 |       : _M_current(__i) { }
      |         ^~~~~~~~~~~~~~~
In static member function ‘static std::unique_ptr<activity_actor> wash_activity_actor::deserialize(JsonIn&)’:
cc1plus: note: source object is likely at address zero
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/fr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/fr/LC_MESSAGES/cataclysm-bn.mo
[  2%] Building CXX object src/CMakeFiles/cataclysm-tiles-common.dir/behavior.cpp.o
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src && /usr/bin/c++ -DBACKTRACE -DCMAKE -DDATA_DIR_PREFIX -DGIT_VERSION -DPREFIX=/usr/local -DRELEASE -DSDL_SOUND -DTILES -DUSE_HOME_DIR -isystem /usr/include/SDL2 -Werror -Wall -Wextra -Wformat-signedness -Wlogical-op -Wmissing-declarations -Wmissing-noreturn -Wnon-virtual-dtor -Wold-style-cast -Woverloaded-virtual -Wpedantic -Wsuggest-override -Wunused-macros -Wzero-as-null-pointer-constant -Wno-unknown-warning-option -Wno-dangling-reference -Wno-range-loop-analysis -Wno-unknown-warning -Wredundant-decls  -fsigned-char -g1 -O3 -DNDEBUG -std=c++17 -pthread -Winvalid-pch -include /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build/src/CMakeFiles/cataclysm-tiles-common.dir/cmake_pch.hxx -MD -MT src/CMakeFiles/cataclysm-tiles-common.dir/behavior.cpp.o -MF CMakeFiles/cataclysm-tiles-common.dir/behavior.cpp.o.d -o CMakeFiles/cataclysm-tiles-common.dir/behavior.cpp.o -c /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/src/behavior.cpp
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/hu.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/hu/LC_MESSAGES/cataclysm-bn.mo
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-range-loop-analysis’ may have been intended to silence earlier diagnostics
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
make[2]: *** [src/CMakeFiles/cataclysm-tiles-common.dir/build.make:157: src/CMakeFiles/cataclysm-tiles-common.dir/activity_actor.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/id.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/id/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/is.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/is/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/it_IT.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/it_IT/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ja.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ja/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ko.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ko/LC_MESSAGES/cataclysm-bn.mo
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make[1]: *** [CMakeFiles/Makefile2:1017: src/CMakeFiles/cataclysm-tiles-common.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/nb.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nb/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/nl.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/nl/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/pl_PL.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pl_PL/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/pt_BR.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/pt_BR/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/ru_RU.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/ru_RU/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/sr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/sr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/tr.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/tr/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/uk.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/uk_UA.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/uk_UA/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/zh_CN.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_CN/LC_MESSAGES/cataclysm-bn.mo
cd /home/rosaaeterna/Documents/GitHub/Cataclysm-BN && /usr/bin/msgfmt -f /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/po/zh_TW.po -o /home/rosaaeterna/Documents/GitHub/Cataclysm-BN/lang/mo/zh_TW/LC_MESSAGES/cataclysm-bn.mo
make[2]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
[  2%] Built target translations_compile
make[1]: Leaving directory '/home/rosaaeterna/Documents/GitHub/Cataclysm-BN/build'
make: *** [Makefile:146: all] Error 2
```

--- a/src/cata_arena.h
+++ b/src/cata_arena.h
@@ -37,12 +37,12 @@ class cata_arena
             return true;
         }
 
-        cata_arena<T>() = default;
-        cata_arena<T>( const cata_arena<T> & ) = delete;
-        cata_arena<T>( cata_arena<T> && ) = delete;
-
+        cata_arena() = default;
 
     public:
+        cata_arena( const cata_arena<T> & ) = delete;
+        cata_arena( cata_arena<T> && ) = delete;
+
         using value_type = T;
 
         static void mark_for_destruction( T *alloc ) {

--- a/src/safe_reference.h
+++ b/src/safe_reference.h
@@ -48,6 +48,7 @@
  */
 
 #include <memory>
+#include <algorithm>
 #include <unordered_map>
 
 #include "debug.h"

